data/content.json

```diff
@@ -1362,6 +1362,27 @@
 				}
 			]
 		},
+		{
+			"name": "TimeSeries",
+			"version": "3.0.0",
+			"widgets": [
+				{
+					"id": "com.mendix.widget.custom.TimeSeries.TimeSeries",
+					"name": "Time series",
+					"description": "Create a time series chart",
+					"docsUrl": "https://docs.mendix.com/appstore/widgets/time-series-chart",
+					"studioCategory": "Charts",
+					"studioProCategory": "Charts",
+					"isPluginWidget": true,
+					"offlineCapable": true,
+					"supportedPlatform": "web",
+					"hasStructureModePreview": false,
+					"hasDesignModePreview": true,
+					"hasAllTileIcons": false,
+					"hasAllDarkIcons": false
+				}
+			]
```

Comment:
@KevinVlaanderen only just noticed it also picks up charts, is it fine to include these altho they aren't published yet?

Reply:
I would say that they do belong in this overview, yea, because it does give information on how complete the widgets are; but it could be interesting to somehow also populate an attribute to show whether it is published or not. Although it sounds like that would have to be maintained manually, which I don't really want to do.

Reply:
Yeah I don't think there's an easy to way to verify that based on our monorepo. We would have to do some API calls/matching with marketplace, which wouldn't be desirable.